README.md

```diff
@@ -1,291 +1,113 @@
+- GitHub: [@wickathou](https://github.com/wickathou)
+- LinkedIn: [@javierjhm](https://linkedin.com/in/javierjhm)
+- Portfolio: [End Design Co.](https://works.enddesign.co/)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- FUTURE FEATURES -->
-
 ## 🔭 Future Features <a name="future-features"></a>
 
 > Describe 1 - 3 features you will add to the project.
 
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
+- [ ] **[Add joins]**
+- [ ] **[Create new tables]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- CONTRIBUTING -->
-
 ## 🤝 Contributing <a name="contributing"></a>
 
-Contributions, issues, and feature requests are welcome!
-
-Feel free to check the [issues page](../../issues/).
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- SUPPORT -->
+Feel free to check the [issues page](https://github.com/wickathou/analytics-reporting/issues).
 
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
-
-If you like this project...
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- ACKNOWLEDGEMENTS -->
+If you like this project feel free to fork it or star it
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
+Thanks to the original creator of the design [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) 
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- FAQ (optional) -->
+Thanks Microverse, learn how to code > [Join Microverse](https://www.microverse.org/?grsf=9m3hq6)
 
 ## ❓ FAQ <a name="faq"></a>
 
 > Add at least 2 questions new developers would ask when they decide to use your project.
 
-- **[Question_1]**
-
-  - [Answer_1]
+- **[I get `createdb: error: connection to server on socket "/tmp/.s.PGSQL.5432" failed: FATAL:  role "joe" does not exist` error, what can I do to fix it?]**
 
-- **[Question_2]**
-
-  - [Answer_2]
+  - [If you have correctly installed PostgreSQL, then try:
+    - From terminal (bash) run `sudo su postgres`, if it works, you should see a change on your command line, with 'postgres@' appended to your terminal command line
+    - Run `psql`, if it works, you should see `postgres=#` in the command line
+    - Then run `create user 'your-username-here' with password 'your-password';`, it will show the message 'CREATE ROLE'
+    - Lastly, apply superuser role to your new username using the command `alter user 'your-username-here' with superuser`, if it worked you should see the message 'ALTER ROLE'
+    - To check that all is working correcly, run `\du` to see the list of users. You should see your newly created user in the list]
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LICENSE -->
-
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
 
```

Comment:
- [ ] I think it would be better if you fix the broken `[MIT](./LICENSE)` link in your README file to make it more professional as it shows a **404** error page once clicked. And to fix that, you can only create a new file name `LICENSE` in your root directory and get its content from this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)** or rename this `[MIT](./LICENSE)` to `[MIT](./MIT.md)`, then the created file in your directory name it `MIT.md` and use the same content in this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)**

Reply:
Oh dang! I forgot it again! Fixed now :rocket: